src/schema/fields/priceCents.ts

```diff
@@ -0,0 +1,35 @@
+import {
+  GraphQLFieldConfig,
+  GraphQLObjectType,
+  GraphQLInt,
+  GraphQLBoolean,
+} from "graphql"
+import { ResolverContext } from "types/graphql"
+
+export const priceCentsField: GraphQLFieldConfig<any, any> = {
+  type: new GraphQLObjectType<any, ResolverContext>({
+    name: "PriceCents",
```

Comment:
is this more like `PriceRange`? could we use a GraphQL Union here? something like:
```graphql
priceCents {
  ... on ExactPrice {
    value
  }
  ... on Range {
    min
    max
  }
}
```

Reply:
I tried this out, have a look and let me know what you think.